test/disability-benefits/components/StatusPage.unit.spec.jsx

```diff
@@ -101,4 +102,59 @@ describe('<StatusPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
+  it('should render notification', () => {
+    const claim = {};
+
+    const tree = SkinDeep.shallowRender(
+      <StatusPage
+          loading
+          message={{ title: 'Test', body: 'Body' }}
+          claim={claim}/>
+    );
+    expect(tree.props.message).not.to.be.null;
```

Comment:
Yep, nice catch. I meant to test that the message is passed to the ClaimDetailLayout component. That component renders the actual notification, so I wasn't going to test that the proper info was being passed to it.